src/components/Text.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default function Text({ 
+  text = "",
+  italic = false, 
+  bold = false, 
+  underline = false, 
+  size = 'text-base', // uses tailwind utility class to set 1rem
+  color = 'text-black', 
+}) { 
+    const classNames = 
+    (italic? 'italic ' : '') + 
+    (bold? 'font-bold ' : '') + 
+    (underline? 'underline ' : '') + 
```

Comment:
Italic, bold and underline are booleans that might used in other places, perhaps think about abstracting this to a utility function that handles Tailwind classname resolution for these specifically

Reply:
I personally feel like what we have here is fine (I'm also unable to figure out other places you'd use italics, bolds, and underlines beyond a text component)